docs/SERVER_CONFIGURATIONS.md

```diff
@@ -561,20 +561,40 @@ creating the `ExternalEventDef`.
 
 ## Monitoring
 
-### `LHS_PROMETHEUS_EXPORTER_PORT`
+### `LHS_HEALTH_SERVICE_PORT`
 
-The port to scrape metrics from.
+The port that the healthchecks and prometheus metrics are exposed on.
 
 - **Type:** int
 - **Default:** 1822
 - **Importance:** low
 
 ---
 
-### `LHS_PROMETHEUS_EXPORTER_PATH`
+### `LHS_HEALTH_PATH_PROMETHEUS`
```

Comment:
LHS_HEALTH_PATH_PROMETHEUS maybe LHS_HEALTH_PATH_METRICS?

Reply:
Ok I can change it. Will change on the Operator too.